pom.xml

```diff
@@ -4,20 +4,20 @@
     <parent>
         <groupId>io.debezium</groupId>
         <artifactId>debezium-parent</artifactId>
-        <version>1.5.0-SNAPSHOT</version>
+        <version>1.5.0.Final</version>
```

Comment:
Not sure why these `.Final`s are showing up as diff - they are the same on the target branch.